src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String border = "____________________________________________________________\n";
+        System.out.print(border + " Hello! I'm Duke\n" + " What can I do for you?\n" + border);
+        String command;
+        int size = 0;
+        String temp;
+        String tag = "[ ]";
+        String[] temp2 = new String[2];
+        String[] list = new String[100];
+        while (true) {
+            try {
+                Scanner read = new Scanner(System.in);
+                command = read.nextLine();
+                if (command.equals("bye")) {
+                    break;
+                } else if (command.equals("list")) {
+                    if (size == 0) {
+                        System.out.print(border + "Empty\n" + border);
+                    } else {
+                        System.out.print(border + "Here are the tasks in your list:\n");
+                        for (int i = 0; i < size; i++) {
+                            System.out.println(" " + Integer.toString(i + 1) + "." + list[i]);
+                        }
+                        System.out.print(border);
+                    }
+                    continue;
+                } else if (command.contains("done")) {
+                    String[] arrOfStr = command.split(" ", 2);
+                    temp = list[Integer.valueOf(arrOfStr[1]) - 1].replace('✗', '✓');
+                    list[Integer.valueOf(arrOfStr[1]) - 1] = temp;
+                    System.out.print(border + " Nice! I've marked this task as done:\n");
+                    System.out.println("  " + list[Integer.valueOf(arrOfStr[1]) - 1]);
+                    System.out.print(border);
+                    continue;
+                } else {
+                    if (command.contains("todo")) {
+                        if (command.equals("todo")){
+                            throw new EmptyTodoException();
+                        }
+                        tag = "[T]";
+                        command = command.replaceAll("todo ", "");
+                        list[size] = tag + "[✗] " + command;
+                    } else if (command.contains("deadline")) {
+                        tag = "[D]";
+                        command = command.replaceAll("deadline ", "");
+                        temp2 = command.split(" /by", 2);
+                        list[size] = tag + "[✗] " + temp2[0] + " (by:" + temp2[1] + ")";
+                    } else if (command.contains("event ")) {
+                        tag = "[E]";
+                        command = command.replaceAll("event ", "");
+                        temp2 = command.split(" /at", 2);
+                        list[size] = tag + "[✗] " + temp2[0] + " (at:" + temp2[1] + ")";
```

Comment:
Consider avoiding deep nesting and check if you could extract them out into methods for better readability